packages/react-components/react-alert/src/components/Alert/Alert.types.ts

```diff
@@ -31,9 +31,14 @@ export type AlertProps = ComponentProps<AlertSlots> & {
    * overrides the intent prop
    */
   intent?: 'info' | 'success' | 'error' | 'warning';
+  /**
+   * The appearance of the Alert.
+   * @default 'primary'
+   */
+  appearance?: 'primary' | 'inverted';
 };
 
 /**
  * State used in rendering Alert
  */
-export type AlertState = ComponentState<AlertSlots> & Pick<AlertProps, 'intent'>;
+export type AlertState = ComponentState<AlertSlots> & Pick<AlertProps, 'appearance' | 'intent'>;
```

Comment:
Should we make appearance required in state as we always default to primary if it's not present?

```suggestion
export type AlertState = ComponentState<AlertSlots> & Pick<AlertProps, 'intent'> & Required<Pick<AlertProps, 'appearance'>>;
```